qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -47,9 +50,16 @@ public static void filterDistros() {
         assumeTrue("only archives", distribution.isArchive());
     }
 
+    private static String superuser = "test_superuser";
+    private static String superuserPassword = "test_superuser";
+
     public void test10Install() throws Exception {
         installation = installArchive(sh, distribution());
         verifyArchiveInstallation(installation, distribution());
+        Result result = sh.run(
+            installation.executables().usersTool + " useradd " + superuser + " -p " + superuserPassword + " -r " + "superuser"
+        );
+        assumeTrue(result.isSuccess());
     }
 
```

Comment:
And another one: test that the nodes that errors in generating the certificate do fail the node startup.
I tested, and the easiest way, I think, is to remove the `bcprov-jdk15on-1.64.jar` from the installation.

Reply:
> if auto-configuration runs, but the node fails bootstrap checks, re-running it with the checks fixed should permit the node to start

I can easily trigger a check ( i.e. the allPermissions one ) but I was wondering if you see this as a possible case for a specific check ? Since we're doing https://github.com/elastic/elasticsearch/pull/77231/files#diff-f175f88713b2d9eae9d9b4eb1fb263eb18851d04e3366176a1b3da6cf7709ab0R398 , I don't, but I have nothing against adding such a test

Reply:
I added a test where the config dir is not writable and one where we fail to generate certificates

Reply:
>I can easily trigger a check ( i.e. the allPermissions one ) but I was wondering if you see this as a possible case for a specific check ? 

Reflecting further it does sound unnecessary at the ArchiveTests level.
My reasoning is that the most common way autoconf is not triggered is because the data dir is not empty, in which case we assume a restart. But it is possible that the data dir stays empty after auto-conf already ran and it  should not run again. But there are many ways this is covered already.